src/resources/resources.json

```diff
@@ -4389,6 +4401,14 @@
     "border": {
       "en": "Border",
       "de": "Rahmen"
+    },
+    "confirmationValidationMessage": {
+      "en": "Please confirm that you agree.",
+      "de": "Bitte bestätige, dass du einverstanden bist."
```

Comment:
This and `allRightsReservedConfirmedRequired` have the same text, I guess they could be reduced to one (maybe call it `agreementConfirmation`?)

Reply:
The other one is not used, I deleted it. Thanks for spotting it!